src/mail-app/mail/view/MailViewerToolbar.ts

```diff
@@ -35,6 +37,7 @@ export interface MailViewerToolbarAttrs {
 	mailViewerViewModel?: MailViewerViewModel
 	mails: Mail[]
 	selectNone?: () => void
+	actionApplyMails: () => Promise<readonly Mail[]>
```

Comment:
I think the name is a bit confusing but I'm also unsure how to name it. "lo fi mails to apply actions to" :sweat_smile: 

I'm also afraid that the people will get confused why we have two lists of mails, maybe the other one should be named something like `selectedMails`.